packages/environment-check/src/cli/index.ts

```diff
@@ -150,7 +151,7 @@ export async function cli(): Promise<void> {
         }
         const options = getOptions(cliArgs) || {};
         options.credentialCallback = credentialCallback;
-        const result = await checkEnvironment(options);
+        const result = isAppStudio() ? await checkBASEnvironment(options) : await checkVSCodeEnvironment();
```

Comment:
as mentioned above, I'd keep `checkEnvironment()` as is and move the check done here inside `checkEnvironment`.

Reply:
Done